src/sql/ResourceSelector.tsx

```diff
@@ -118,6 +119,8 @@ export function ResourceSelector(props: ResourceSelectorProps) {
           className={props.className || 'min-width-6'}
           disabled={props.disabled}
           onOpenMenu={() => props.fetch && onClick()}
+          allowCustomValue={props.allowCustomValue}
+          menuShouldPortal={true}
```

Comment:
we had a warning when running the test because of that - we should probably add that to all the other selects too (in another PR!)